languages/Cobol.groovy

```diff
@@ -339,3 +339,17 @@ def getRepositoryClient() {
 
 	return repositoryClient
 }
+
+boolean buildListContainsTests(List<String> buildList) {
+	// iterate through build list
+	buildList.each { buildFile ->
+	
+		// Check if this is a testcase
+		isZUnitTestCase = (props.getFileProperty('cobol_testcase', buildFile).equals('true')) ? true : false
+		
+		if (isZUnitTestCase) {
+			return true
```

Comment:
What I understood from the docs, you cannot break/return out from an each{closure} loop, that is why your current method always returns `false` and the datasets are not allocated.

Let's just switch to a `return isZUnitTestCase` at the very end.

Reply:
@EmanuP 

Can you please replace the method by, I already tested the code. This should be fine:

```
// check build list for test cases
def boolean buildListContainsTests(List<String> buildList) {

	boolean containsZUnitTestCase = buildList.find { buildFile -> props.getFileProperty('cobol_testcase', buildFile).equals('true')}
	
	return containsZUnitTestCase ? true : false
}
```